contracts/validator-manager/ValidatorManager.sol

```diff
@@ -251,9 +248,11 @@ abstract contract ValidatorManager is Initializable, ContextUpgradeable, IValida
             ValidatorMessages.ValidationPeriod({
                 subnetID: $._subnetID,
                 nodeID: input.nodeID,
-                weight: weight,
                 blsPublicKey: input.blsPublicKey,
-                registrationExpiry: input.registrationExpiry
+                remainingBalanceOwner: input.remainingBalanceOwner,
```

Comment:
Should we validate the `PChainOwners` here? The requirements are:
- If threshold is 0, addresses must be empty.
- threshold <= len(addresses)
- addresses must be sorted in ascending order